superflore/generators/ebuild/ebuild.py

```diff
@@ -19,7 +19,7 @@
 from superflore.utils import resolve_dep
 from superflore.utils import sanitize_string
 from superflore.utils import trim_string
-
+from time import gmtime, strftime
```

Comment:
This should be above in the block with the other system imports (just below sys) And leave the white space here.